src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static String getRandomColor() {
+        Random random = new Random();
+        Colors[] colors = Colors.values();
+        return colors[random.nextInt(colors.length)].toString();
```

Comment:
[Checklist ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants) issue - is `toString()` optimal choice for a string representation of enum?